torch/_refs/__init__.py

```diff
@@ -3275,6 +3275,20 @@ def new_empty_strided(
     )
 
 
+@register_decomposition(torch.ops.aten.zeros)
+def zeros(
+    size: ShapeType,
+    *,
+    dtype: Optional[torch.dtype] = None,
+    layout: Optional[torch.layout] = None,
+    device: Optional[torch.device] = None,
+    pin_memory: bool = False,
+) -> TensorLikeType:
+    return torch.full(
+        size, 0, dtype=dtype, layout=layout, device=device, pin_memory=pin_memory
+    )
+
+
 @register_decomposition(torch.ops.aten.new_zeros)
 def new_zeros(
```

Comment:
What about the `requires_grad` parameter?

Reply:
(If the ATen version doesn't support `requires_grad` but the torch version does then let's at least add a comment to that effect -- we'll probably want to separate references that are different between ATen and torch soon)

Reply:
added.